src/developer-tools/get-started/manage-accounts.html

```diff
@@ -7,8 +7,12 @@
---request GET 'https://xpring.io/portal/ilp/hermes/accounts/{your-account-id}/balance' \
+        <div class="editable"><h1>Manage Interledger Accounts Programmatically</h1>
+            <p>Make payments programmatically on a testnet.</p>
+            <p><em>Before starting this tutorial, make sure you have created accounts in both the Rafiki by following the instructions in the&nbsp;<a target="_blank" href="https://interledger.org/setup-wallets.html">Setup Wallets&nbsp;<em></em></a>&nbsp;tutorial.</em></p>
+            <p><em>In addition, make sure you have created a programmable account in the RippleX Faucet at <a target="_blank" href="https://faucet.ilpv4.dev">ILPv4 Faucet</a>. Note that the RippleX wallet does not currently support programmability, so credentials from the ilpv4.dev faucet must be used instead.</p>
+            <p>To get started using Interledger, this tutorial helps you access the Interledger testnet programmatically. This Interledger testnet is currently maintained by teams at&nbsp;<a target="_blank" href="https://ripplex.io/">RippleX&nbsp;<em></em></a>&nbsp;and&nbsp;<a target="_blank" href="https://coil.com/">Coil</a>, but anyone can join. Note that all Interledger test networks use fake money, rather than transferring real value.</p><p>The RippleX and Rafiki testnet connectors, both used in this tutorial, together form a network for users to practice sending and receiving test currency values. Although this tutorial uses test XRP for simplicity, Interledger can handle transactions involving any currency pairs, as described in&nbsp;<a href="https://interledger.org/overview.html">Interledger Overview</a>.</p><p>In this tutorial, you will:</p><ol><li><a href="https://interledger.org/setup-wallets-programmatically.html#1-check-your-balance">Check Your Balance</a>.</li><li><a href="https://interledger.org/setup-wallets-programmatically.html#2-pay-a-friend">Pay a Friend</a>&nbsp;to another payment pointer.</li><li><a href="https://interledger.org/setup-wallets-programmatically.html#3-get-paid">Get Paid</a>.</li></ol><h3>1. Check Your Balance</h3><p>To programmatically see how much money is in your account, use the following&nbsp;<a target="_blank" href="https://curl.haxx.se/docs/manual.html">curl&nbsp;<em></em></a>&nbsp;call. Be sure to replace&nbsp;<strong><code>{your-account-id}</code></strong>&nbsp;and&nbsp;<strong><code>{auth_token}</code></strong>&nbsp;above with the values obtained from the <a target="_blank" href="https://faucet.ilpv4.dev/">ILP Faucet portal</a>.</p><pre><code>curl --location
```

Comment:
```suggestion
            <p>To get started using Interledger, this tutorial helps you access the Interledger testnet programmatically. This Interledger testnet is currently maintained by teams at&nbsp;<a target="_blank" href="https://ripple.com/ripplex/">RippleX&nbsp;<em></em></a>&nbsp;and&nbsp;<a target="_blank" href="https://coil.com/">Coil</a>, but anyone can join. Note that all Interledger test networks use fake money, rather than transferring real value.</p><p>The RippleX and Rafiki testnet connectors, both used in this tutorial, together form a network for users to practice sending and receiving test currency values. Although this tutorial uses test XRP for simplicity, Interledger can handle transactions involving any currency pairs, as described in&nbsp;<a href="https://interledger.org/overview.html">Interledger Overview</a>.</p><p>In this tutorial, you will:</p><ol><li><a href="https://interledger.org/setup-wallets-programmatically.html#1-check-your-balance">Check Your Balance</a>.</li><li><a href="https://interledger.org/setup-wallets-programmatically.html#2-pay-a-friend">Pay a Friend</a>&nbsp;to another payment pointer.</li><li><a href="https://interledger.org/setup-wallets-programmatically.html#3-get-paid">Get Paid</a>.</li></ol><h3>1. Check Your Balance</h3><p>To programmatically see how much money is in your account, use the following&nbsp;<a target="_blank" href="https://curl.haxx.se/docs/manual.html">curl&nbsp;<em></em></a>&nbsp;call. Be sure to replace&nbsp;<strong><code>{your-account-id}</code></strong>&nbsp;and&nbsp;<strong><code>{auth_token}</code></strong>&nbsp;above with the values obtained from the <a target="_blank" href="https://faucet.ilpv4.dev/">ILP Faucet portal</a>.</p><pre><code>curl --location
```